lib/node/pl-errors/src/parsed_error.ts

```diff
@@ -247,6 +247,9 @@ export function parseSubErrors(message: string): PlCoreError[] {
       // we need initial stage because apparently the first line
       // of the error doesn't have [I], but is a path line.
       state.stage = 'path';
+    } else if ( line.startsWith('---')) {
```

Comment:
Looks good! Do you think there could be programs that also have lines with `---` though? Maybe we should change it to a more unique set of characters, like 10 random chars, for example

Reply:
Or, another variant (since `---` looks good), is to check the separator only when we are parsing `path` part of the message (`state.stage == 'path'`)